### PR TITLE
docs: Update PostgreSQL CDC to Snowflake tutorial with technical details.

### PR DESCRIPTION
**Pull request by danthelion**
**Description:**

Update the Tutorial based on comments from @willdonnelly. Thanks for the review!

**Documentation links affected:**

[The tutorial](https://docs.estuary.dev/getting-started/tutorials/postgresql_cdc_to_snowflake/)

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/flow/1454)

### CODE REVIEW

**Review by willdonnelly — APPROVED** [review]
Looks good! A few more nitpicks, although only the question about captures having update delays is an actual accuracy issue, the other comments are just places where it seemed like the wording could potentially be made clearer.

File: site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md [hunk 1]
```diff
@@ -374,6 +386,16 @@ You can grab your Snowflake host URL and account identifier by navigating to the
 
 ![Grab your Snowflake account id](https://storage.googleapis.com/estuary-marketing-strapi-uploads/uploads//snowflake_account_id_af1cc78df8/snowflake_account_id_af1cc78df8.png)
 
+If you scroll down to the Advanced Options section, you will be able to configure the "Update Delay" parameter. If you leave this parameter unset, the default value of 30 minutes will be used.
+
+![Update Delay](https://storage.googleapis.com/estuary-marketing-strapi-uploads/uploads//snowflake_update_delay_dark_f26179d3fc/snowflake_update_delay_dark_f26179d3fc.png)
+
+The Update Delay parameter in Estuary materializations offers a flexible approach to data ingestion scheduling. This option allows users to control when the materialization or capture tasks pull in new data by specifying a delay period.
```
[the remark this answers is not in the document]
Comment (danthelion) [reply site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md]: I think they are indeed specific for dwh's, updating!

File: site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md [hunk 2]
```diff
@@ -374,6 +386,16 @@ You can grab your Snowflake host URL and account identifier by navigating to the
...
+
+It represents the amount of time the system will wait before it begins materializing the latest data.
+
+For example, if an update delay is set to 2 hours, the materialization task will pause for 2 hours before processing the latest available data. This delay ensures that data is not pulled in immediately after it becomes available, allowing for batching and other optimizations that can reduce warehouse load and processing time.
```
[the remark this answers is not in the document]
Comment (danthelion) [reply site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md]: Added!

File: site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md [hunk 3]
```diff
@@ -390,7 +412,12 @@ And that’s pretty much it, you’ve successfully published a real-time CDC pip
...
 
-Looks like the data is arriving as expected, and the schema of the table is properly configured by the connector based on the types of the original table in Postgres. To get a feel for how low latency the pipeline is, feel free to jump back into the terminal and insert or modify some records in our products table then head over to either the Flow web UI or straight to Snowflake and query the materialized dataset to see the changes!
+Looks like the data is arriving as expected, and the schema of the table is properly configured by the connector based on the types of the original table in Postgres.
+
+To get a feel for how the data flow works; feel free to jump back into the terminal and insert or modify some records in our products table then head over to either the Flow web UI or to Snowflake and query the materialized dataset to see the changes!
+
+:::note Based on your configuration of the "Update Delay" parameter when setting up the Snowflake Materialization, you might have to wait until the configured amount of time passes in order for the Materialization process to be triggered.
```
[the remark this answers is not in the document]
Comment (danthelion) [reply site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md]: Fair! Updated language.

File: site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md [hunk 4]
```diff
@@ -390,7 +412,12 @@ And that’s pretty much it, you’ve successfully published a real-time CDC pip
 
 ![Results in Snowflake](https://storage.googleapis.com/estuary-marketing-strapi-uploads/uploads//snowflake_verification_2eb047efec/snowflake_verification_2eb047efec.png)
 
-Looks like the data is arriving as expected, and the schema of the table is properly configured by the connector based on the types of the original table in Postgres. To get a feel for how low latency the pipeline is, feel free to jump back into the terminal and insert or modify some records in our products table then head over to either the Flow web UI or straight to Snowflake and query the materialized dataset to see the changes!
+Looks like the data is arriving as expected, and the schema of the table is properly configured by the connector based on the types of the original table in Postgres.
+
+To get a feel for how the data flow works; feel free to jump back into the terminal and insert or modify some records in our products table then head over to either the Flow web UI or to Snowflake and query the materialized dataset to see the changes!
```
[the remark this answers is not in the document]
Comment (danthelion) [reply site/docs/getting-started/tutorials/postgresql_cdc_to_snowflake.md]: You're totally right, I mixed the two unnecessarily here. Updated.

**Pull request merged**